### DISCUSSION TITLE
Welcome to bwalink Discussions!

### DISCUSSION THREAD

**Discussion by jshank** [jshank/bwalink#26] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KA4HRE** [comment]
This works great on my BWA system in my swim spa!  The EW11 works!  The Home Assistant stuff via MQTT works!  I could NOT have done this without this GitHub page - THANK YOU!!!!!

**Comment by StKi1da** [comment]
Going through this set up for my hot tub. I have connected the EW-11A which is configured. I already have a pi running home assistant. Forgive my ignorance - but do I need a different raspberry pi running rasbian (or perhaps ubuntu?) and docker so the hot tub passes the messages to rasbian pi which then publishes the mqtt which home assistant is listening for? I initially tried to run using the addon instructions but couldnt get that to work so reverted to docker and portainer (using stack for the docker-compose file) - but this gives a "exec /docker-entrypoint.sh: exec format error" does that mean the raspberry pi5 chip is not compatible?

**Comment by KA4HRE** [comment]
I did not need anything additional to my already existing RPi 4 running HA.

Here is what my MQTT device it detected looks like if this helps:

![image](https://github.com/user-attachments/assets/ecef9175-8b7c-44d6-b86e-259a4d46f72c)

**Reply by StKi1da** [reply]
Thanks for your reply. So on the rpi4 you installed HAOS first then docker /docker-compose afterwards and only the bwalink in docker?

**Comment by KA4HRE** [comment]
I didn't install any docker anything.  My HA runs on the RPi4 standalone.
IIR, the instructions include an option for doing it without the need for a
docker if you have HA running already ?!?
Sorry if I'm not explaining this well...

**Reply by StKi1da** [reply]
Decided to go back to first principles and reinstalled HA afresh - and quickly got the hot tub data working. I've had the previous HA instance running quite a while now and am thinking something has done something along the way! Probably a good thing for the refresh!

**Comment by StKi1da** [comment]
Yeah. I tried that but was getting an error on a rpi5 but  will try a rpi4
as the error code looked chip related.

**Comment by KA4HRE** [comment]
Great to hear!

**Comment by maninsheds** [comment]
Hi 

I'm trying to install the bwalink HA addon, but I get the following error when I click install in the addon store:

The command '/bin/ash -o pipefail -c apk --no-cache add ruby ruby-io-console ruby-irb ruby-json ruby-etc ruby-bigdecimal ruby-rdoc libffi-dev zlib-dev ruby-bundler' returned a non-zero code: 4

Can anyone help?

Running HAOS on a VM if that makes any difference?

Thanks

Dan
